Lib/email/_policybase.py

```diff
@@ -307,7 +307,29 @@ def header_source_parse(self, sourcelines):
 
         """
         name, value = sourcelines[0].split(':', 1)
-        value = value.lstrip(' \t') + ''.join(sourcelines[1:])
+
+        # Fixed: https://github.com/python/cpython/issues/124452
+        #
+        # Root cause: The function '_refold_parse_tree' in '_header_value_parse.py'.
+        # If there is no WSP, it can't figure out how to wrap the text.
+        # Therefore, it places the entire value directly after '\n', and because
+        # there is a WSP after '<HeaderName>:', the WSP will be moved to the front
+        # of the value according to RFC5322, section 2.2.3.
+        #
+        # However, the WSP is not part of the value; therefore, we must
+        # remove it.
```

Comment:
I would simply keep the reference to the issue without restating the issue itself.

Reply:
Hmm, it's useful to me when there's a comment explaining at least partially what the issue was. Maybe it could be shortened though.

Reply:
I would not add any comment, since the fix makes the code better match the intent of the docstring (see below for the docstring revision).

By the way, long explanations like what you wrote should *definitely* be included in the commit message.